.ci-integration-tests.yml

```diff
@@ -0,0 +1,10 @@
+pipeline_template: ci/modules/Jenkinsfile-integration-tests
+lock: terraform-module-ci-lab
+env_vars:
+    INMANTA_LSM_USER: rocky
```

Comment:
Is this one required? I don't think the terraform module uses lsm.

Reply:
it's not, I forgot to remove it, thanks :p